playbooks/roles/edxapp/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
   register: bash_theme
 
 # Print the shell tasks's stdout and stderr.
-- debug: var="bash_theme.stdout_lines + [ bash_theme.stderr ]"
+#- debug: var="bash_theme.stdout_lines + [ bash_theme.stderr ]"
```

Comment:
Why are you commenting out theming debug code? Isn't that helpful for troubleshooting issues?